verifier/sources/verifier_channel.move

```diff
@@ -0,0 +1,244 @@
+module verifier_addr::verifier_channel {
+    use std::vector::{append, borrow, borrow_mut, enumerate_ref, slice};
+    use aptos_std::aptos_hash::keccak256;
+
+    use lib_addr::bytes::{bytes32_to_u256, num_to_bytes_le, vec_to_bytes_le};
+    use lib_addr::prime_field_element_0::{fmul, from_montgomery};
+    use lib_addr::vector::set_el;
+    use verifier_addr::prng::{get_random_bytes, init_prng};
+
+    // This line is used for generating constants DO NOT REMOVE!
+    // 3
+    const EMASK_MUST_BE_LESS_THAN_2_TO_THE_POWER_OF_64: u64 = 0x3;
+    // 100
+    const EOVERFLOW_PROTECTION_FAILED: u64 = 0x64;
+    // 2
+    const EPROOF_OF_WORK_CHECK_FAILED: u64 = 0x2;
+    // 113078212145816603762751633895895194930089271709401121343797004406777446400
+    const K_MONTGOMERY_R_INV: u256 = 0x40000000000001100000000000012100000000000000000000000000000000;
+    // End of generating constants!
+
+    friend verifier_addr::stark_verifier_7;
+    friend verifier_addr::fri_statement_verifier_7;
+
+    public(friend) inline fun get_prng_ptr(channel_ptr: u64): u64 {
```

Comment:
remove this function

Reply:
what's wrong with it?

Reply:
only used once ?

Reply:
Calling this function makes more sense than using `channel_ptr + 1` directly

Reply:
Then replace all  `channel_ptr + 1` by `get_prng_ptr`. Anyway, don't use this function